security/app-layer-policy.md

```diff
@@ -38,18 +38,16 @@ See [Enforce network policy using Istio tutorial]({{site.baseurl}}/security/tuto
 
 - [{{site.prodname}} is installed]({{site.baseurl}}/getting-started/kubernetes/)
 - [calicoctl is installed and configured]({{site.baseurl}}/getting-started/clis/calicoctl/install)
-- Kubernetes 1.15 or older (Istio 1.1.7 does not support Kubernetes 1.16+).
-See this [issue](https://github.com/projectcalico/calico/issues/2943){:target="_blank"} for details and workaround.
 
 **Istio support**
 
 Following Istio versions have been verified to work with application layer policies:
-- Istio v1.7.4
-- Istio v1.6.13
+- Istio v1.10.2
+- Istio v1.9.6
 
-Istio v1.5.x is **not** supported.
+Istio v1.6.x and lower are **not** supported.
```

Comment:
Istio v1.6 and older use Envoy v2 by default and I don't think we want to support that anymore. I haven't verified Istio 1.7 or 1.8 but 1.7 could still be working. We don't have any manifests for Istio 1.8 so that would require separate work to make that work.

Reply:
sounds good. I think your approach is identical to what we've done in the past.